scripts/history/utils/index.js

```diff
@@ -0,0 +1,23 @@
+import fsApi from 'fs';
+
+const fs = fsApi.promises;
+
+export async function importReadme({ from: sourceRepository, to: targetRepository }) {
+  const sourceRepositoryReadmePath = `${sourceRepository.path}/README.md`;
+  const targetRepositoryReadmePath = `${targetRepository.path}/README.md`;
+
+  const [readmeCommit] = await sourceRepository.git.log(['README.md']);
```

Comment:
This expects that only a single commit contains the README, which does not sound right to me. I see that you then copy the file contents and get just the message and date of the first commit, which could be understood —if I am right— as squashing all README commits to the first one. This is acceptable behaviour to me but would benefit from being made more explicit.
```suggestion
  const [firstReadmeCommit] = await sourceRepository.git.log(['README.md']);
```